contractcourt/chain_arbitrator.go

```diff
@@ -697,6 +711,33 @@ func (c *ChainArbitrator) Start(beat chainio.Blockbeat) error {
 	return nil
 }
 
+// resolveContracts listens to the `resolvedChan` to mark a given channel as
+// fully resolved.
+func (c *ChainArbitrator) resolveContracts() {
+	// Consume block epochs until we receive the instruction to shutdown.
```

Comment:
Nit: Are we consuming block epochs here? Seems to me we only listen to the resolvedChan ?